### PR TITLE
Add Venda and Tsonga language codes

### PR DESCRIPTION
**Pull request by eatyourgreens**
Labels and ISO 693-3 language codes for Tshivenḓa and Xitsonga.

Staging branch URL: https://pr-6729.pfe-preview.zooniverse.org

### CODE REVIEW

File: app/constants/languages.js [hunk 1]
```diff
@@ -118,6 +119,7 @@ const languages = [
   { value: 'vi', label: 'Vietlabelse' },
   { value: 'cy', label: 'Welsh' },
   { value: 'wo-sn', label: 'Wolof (Senegal)' },
+  { value: 'tso', label: 'Xitsonga'},
```
Comment (eatyourgreens) [inline app/constants/languages.js:122]:
```suggestion
  { value: 'ts', label: 'Xitsonga'},
```

**Review by eatyourgreens — COMMENTED** [review]
Should we prefer 2-letter codes, where they exist?

File: app/constants/languages.js [hunk 2]
```diff
@@ -118,6 +119,7 @@ const languages = [
   { value: 'vi', label: 'Vietlabelse' },
   { value: 'cy', label: 'Welsh' },
   { value: 'wo-sn', label: 'Wolof (Senegal)' },
+  { value: 'ts', label: 'Xitsonga'},
```
Comment (lcjohnso) [inline app/constants/languages.js:122]:
```suggestion
  { value: 'ts', label: 'Tsonga'},
```

File: app/constants/languages.js [hunk 3]
```diff
@@ -108,6 +108,7 @@ const languages = [
   { value: 'te', label: 'Telugu' },
   { value: 'th', label: 'Thai' },
   { value: 'bo-cn', label: 'Tibetan (PRC)' },
+  { value: 've', label: 'Tshivenḓa' },
```
Comment (lcjohnso) [inline app/constants/languages.js:111]:
```suggestion
  { value: 've', label: 'Venda' },
```

**Review by lcjohnso — APPROVED** [review]
Ok, but I prefer English names/labels that match reference lists (those from [LOC](https://www.loc.gov/standards/iso639-2/php/code_list.php) or [Lokalise](https://docs.lokalise.com/en/articles/1400544-language-settings)).

**Comment by eatyourgreens** [comment]
The names here are the language names used by our South African colleagues, so I’ll leave the choice to them.

**Pull request merged**